components/scream/cmake/Kokkos.cmake

```diff
@@ -0,0 +1,25 @@
+# Allow a few variants for specifying the kokkos installation directory
+if (Kokkos_DIR)
+  SET (KOKKOS_INSTALL_DIR ${Kokkos_DIR})
+elseif (KOKKOS_DIR)
+  SET (KOKKOS_INSTALL_DIR ${KOKKOS_DIR})
+else ()
+  message (FATAL_ERROR "SCREAM requires Kokkos. Set one of Kokkos_DIR or KOKKOS_DIR to point to the base directory of the Kokkos installation.")
+endif ()
+
+# Set some kokkos variables
+include (${KOKKOS_INSTALL_DIR}/kokkos_generated_settings.cmake)
+set (Kokkos_INCLUDE_DIR ${KOKKOS_INSTALL_DIR}/include)
+set (Kokkos_LIBRARY_DIR ${KOKKOS_INSTALL_DIR}/lib)
+set (Kokkos_LINK_FLAGS ${KOKKOS_LINK_FLAGS})
+
+# LB: is this really needed?
```

Comment:
Yes, you will see errors on CUDA systems without this loop.

Reply:
@bartgol - Ok, your plan sounds good.